### PR TITLE
Hotfixes v2 17

### PR DESCRIPTION
**Pull request by JamieWeider72**

### CODE REVIEW

File: docs/home/changelog/hotfixes-2-16.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ date: 2024-Feb-26
 
 The following is a list of the known and fixed issues for Run:ai V2.16.
 
+
+## Version 2.16.25
+
+| Internal ID | Description |
+|--|--|
+| RUN-17241 | Fixed an issue where the nodes page showed nodes as not ready to to "tookit not installed". |
```
Comment (ozRunAI) [inline docs/home/changelog/hotfixes-2-16.md:16]:
```suggestion
| RUN-17241 | Fixed an issue where the nodes page showed nodes as not ready due to "tookit not installed". |
```

File: docs/home/whats-new-2-18.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
+
+* <!-- TODO ADD LINK TO DOC Run-14732/Run-14733 Add backoff limit to workspace & standard training -->Added backoff limit functionality to Training and Workspace workloads. The backoff limit is the maximum number of retry attempts for failed workloads. After reaching the limit, the workload's status will change to `Failed`.
```
Comment (ozRunAI) [inline docs/home/whats-new-2-18.md:18]: TODO placeholder?

File: docs/home/whats-new-2-18.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
```
Comment (ozRunAI) [inline docs/home/whats-new-2-18.md:16]: Should we keep using Jobs / Workspaces? or only "Workloads"?

File: docs/home/whats-new-2-18.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: Version 2.17
```
Comment (ozRunAI) [inline docs/home/whats-new-2-18.md:2]:
```suggestion
title: Version 2.18
```

**Review by ozRunAI — CHANGES_REQUESTED** [review]

File: docs/home/whats-new-2-18.md (hunk 3 above)
Comment (jasonnovichRunAI) [reply docs/home/whats-new-2-18.md:16]: @ozRunAI IMHO I think we still need to keep *Jobs* until it is EOL. As long as it still Deprecated, then we need to keep it alive. Another consideration could be that k8s call these jobs. So we need to make clear are w referencing **Run:ai Jobs** or **k8s jobs**.

**Review by jasonnovichRunAI — APPROVED** [review]

**Pull request merged**